server/channels/app/server.go

```diff
@@ -996,6 +996,11 @@ func (s *Server) Start() error {
 	logListeningPort := fmt.Sprintf("Server is listening on %v", listener.Addr().String())
 	mlog.Info(logListeningPort, mlog.String("address", listener.Addr().String()))
 
+	// Check if the command is called by root user
+	if os.Geteuid() == 0 {
+		mlog.Warn("Running as root is not recommended")
+	}
```

Comment:
Let's move that code closer to other sanity checks. Next to https://github.com/mattermost/mattermost/blob/37da0531919032722dfc8036b293bd1bdc900a7b/server/channels/app/server.go#L415-L417 would be a good place.